common/resources.yaml

```
@@ -57,9 +51,15 @@ groups:
     name: objects
     doc: A table for identifying which objects in a file contain references to external resources.
     dtype:
+    - name: file_idx
```

```suggestion
    - name: files_idx
```
Sorry to request another change. Can we make this `files_idx` to be consistent with the other `idx` names?